openfisca_uk/parameters/calibration/budgetary_impact/by_program.yaml

```diff
@@ -6,6 +6,10 @@ JSA_income:
     2020-01-01: 435_000_000
     2021-01-01: 297_000_000
     2022-01-01: 196_000_000
+    2023-01-01: 21_000
```

Comment:
This is a linear extrapolation right? How realistic are these `JSA_income` values?

Reply:
Actually this and other projections are projections from the DWP

Reply:
I'm still seeing 196m and 21m, from here https://view.officeapps.live.com/op/view.aspx?src=https%3A%2F%2Fassets.publishing.service.gov.uk%2Fgovernment%2Fuploads%2Fsystem%2Fuploads%2Fattachment_data%2Ffile%2F1034496%2Foutturn-and-forecast-tables-autumn-budget-2021.ods&wdOrigin=BROWSELINK